pkg/util/logging/logger.go

```diff
@@ -22,7 +22,6 @@ import (
 	"os"
 
 	logging "github.com/op/go-logging"
-	"github.com/openshift/ansible-service-broker/pkg/config"
 )
```

Comment:
The package of this file is `package app` yet it lives in `util/logging` which is the right package?